### ISSUE TITLE
Can't save more than 4 groups

### ISSUE THREAD

**Issue by yegorius** [hharnisc/tabbie#28]
Hello,
thank you for this great extension. It fits my browsing habits very well.
I have a lot of browser windows which I use like sessions but I can't save more than 4 of them.
Excuse me for the image, it's from HiDPI screen.
![img-2017-01-10-174727](https://cloud.githubusercontent.com/assets/309959/21812831/f264eefc-d75c-11e6-88e3-2d34e3b203c7.png)

**Comment by hharnisc** [comment]
Hey @yegorius, it's definitely not clear at the moment but you _should_ be able to scroll that list of tab groups if there are more.  Please let me know if that's not the case though, there could be something else going on.

**Comment by yegorius** [comment]
Yes, it feels like I can create additional group, but I can't see it and can't scroll to it. This could be a HiDPI screen issue.

**Comment by hharnisc** [comment]
Might be HiDPI, could also be something else. Would you be up for sharing a screenshot of the Tabbie console here?

- Right click on the Tabbie icon
- Click on `inspect popup`
- Take a screenshot of the developer console window that appears

Would be interested in seeing if there are any errors there.

**Comment by yegorius** [comment]
Oh I see. The problem shows up only with one particular window with a lot of tabs. Other windows can be saved with no issues.
![img-2017-01-10-183130](https://cloud.githubusercontent.com/assets/309959/21814789/2416f4c6-d763-11e6-875c-b998e8cf61e2.png)

**Comment by hharnisc** [comment]
Oh my! You're the first person to report this, and I had a feeling it would happen eventually.

The chrome sync API which Tabbie uses has a limit. There's enough tabs saved here to reach that limit. 

If you don't mind me asking, could you describe how many tabs you've got open in each tab group?

**Comment by hharnisc** [comment]
I'll need to figure out the best way to handle this too, currently the tabs will be synced across browsers. Is that something you would use?

**Comment by yegorius** [comment]
There are 23 tabs in that window. Tab groups sync would be a killer feature. Right now I use the same laptop at home and at work, but I used to have a desktop PC which I sold because I couldn't sync tab groups efficiently :)

**Comment by hharnisc** [comment]
@yegorius unrelated, are you using i3 for your manager with ubuntu? What hardware are you using BTW?

**Comment by hharnisc** [comment]
Still thinking about how best to solve this, I could add a setting to use local storage so you could save a lot more data. You'd lose the ability to sync these between machines though.

Or I've been thinking about creating a backend to store the data, so you'd be able to sync as many machines as needed. I'd have to host it somewhere so there would be a cost involved with that. Is this a feature you'd be willing to pay for if it worked well?

**Comment by anirbanroydas** [comment]
Having the exact same issue. I use chrome too heavily. Sometimes 6-8 windows summing upto 150-250 tabs for some crazy research stuffs. And hence unable to save those tabs. 

Moreover developer console shows the same error.

**Comment by yegorius** [comment]
Right now I don't need the sync feature because I use the same laptop at work and at home. I would rather rely on local storage in my case. But I think tab groups sync is a great feature and other people might need that. 
Offtopic: I use i3 with ArchLinux on Dell XPS 15 9550 and 4K screen

**Comment by yegorius** [comment]
One possible solution could be like this:
1. Give an option to the user to select storage backend:
a) local, no sync, unlimited space
b) chrome internal store, with sync, limited space
2. Fail fast. If a user selects internal storage and saves too many tabs, such that QUOTA_BYTES_PER_ITEM exception is raised, show a warning immediately, so that user can save tabs by other means (because I used to loose my tabs a few times this way).

**Comment by kurtextrem** [comment]
Same issue here, but no errors. I can save the tabs and they'll be added to the list, but when I open Tabbie again the entry is gone.

**Comment by kurtextrem** [comment]
"Quick" fix:
- Download my fork: https://github.com/kurtextrem/tabbie
- Unzip
- In Chrome, go to `chrome://extensions`, enable dev mode and click Load unpacked extension - select folder `dist`

**Comment by kurtextrem** [comment]
Alright guys, here we go, I've published updated Tabbie:  https://chrome.google.com/webstore/detail/eiigehbpamofgandbmmokoahhkhbmbld/
